Makefile

```diff
@@ -126,6 +126,7 @@ clean:
 
 
 KOCACHE_PATH=/tmp/ko
+KOARTIFACT_HUB=--image-label io.artifacthub.package.readme-url=https://raw.githubusercontent.com/sigstore/cosign/main/README.md --image-label io.artifacthub.package.logo-url=https://raw.githubusercontent.com/sigstore/cosign/main/images/logo.svg --image-label io.artifacthub.package.license=Apache-2.0 --image-label io.artifacthub.package.vendor=sigstore --image-label io.artifacthub.package.version=0.1.0 --image-label io.artifacthub.package.name=cosign --image-label org.opencontainers.image.created=$(BUILD_DATE) --image-label org.opencontainers.image.description='Container signing verification and storage in an OCI registry'
```

Comment:
nit, maybe call this "ARTIFACT_HUB_LABELS" to make it bit more clear what they're for, and to disambiguate it from the built-in ko vars.